### PR TITLE
Teet 94

### PR DESCRIPTION
**Pull request by kuroneko1011**

### CODE REVIEW

File: src/components/customized-components/SearchAccountFormComponent/SearchAccountForm.component.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        </Col>
+
+        <Col span={2}>
+          <Form.Item>
+            <Button type="primary" htmlType="submit">
+              Search
+            </Button>
+          </Form.Item>
+        </Col>
+        {/* <Col span={2}>
```
Comment (ZERO-94) [inline src/components/customized-components/SearchAccountFormComponent/SearchAccountForm.component.jsx:53]: why comment this ?

File: src/components/customized-components/SearchAccountFormComponent/SearchAccountForm.component.jsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { Form, Col, Row, Input, Button, Select } from "antd";
+import { RoleConstantArray } from "../../../constants/RoleConstants";
+import { AccountStatusConstantsArray } from "../../../constants/AccountStatusConstants";
+
+const SearchAccountForm = ({ resetAction, ...antdProps }) => {
```
Comment (ZERO-94) [inline src/components/customized-components/SearchAccountFormComponent/SearchAccountForm.component.jsx:6]: is this worth to put in separate file like this ? or only need to put this function in the same table container file ?

File: src/constants/AccountStatusConstants.js [hunk 3]
```diff
@@ -1,7 +1,11 @@
-const AccountStatusConstants = {
+export const AccountStatusConstants = {
   ACTIVE: "ACTIVE",
   INACTIVE: "INACTIVE",
   PENDING: "PENDING",
 };
 
-export default AccountStatusConstants;
+export const AccountStatusConstantsArray = [
```
Comment (ZERO-94) [inline src/constants/AccountStatusConstants.js:7]: remove this

File: src/constants/RoleConstants.js [hunk 4]
```diff
@@ -2,6 +2,14 @@ export const RoleConstant = {
   MANAGER: "MANAGER",
   CUSTOMER_SERVICE: "CUSTOMER_SERVICE",
   DENTIST: "DENTIST",
-  CUSTOMER: "CUSTOMER",
+  PATIENT: "PATIENT",
   ADMIN: "ADMIN",
 };
+
+export const RoleConstantArray = [
```
Comment (ZERO-94) [inline src/constants/RoleConstants.js:9]: remove this

File: src/containers/AccountManagementTable/AccountManagementTable.container.jsx [hunk 5]
```diff
@@ -1,23 +1,59 @@
...
+    setFilterData({
+      id: values.id,
+      role: values.role,
+      status: values.status,
+      fullName: values.fullName,
+    });
+  };
+
+  const fetchData = async (options) => {
+    console.log("a");
```
Comment (ZERO-94) [inline src/containers/AccountManagementTable/AccountManagementTable.container.jsx:46]: remove this

File: src/containers/CustomerServiceCreateForm/CustomerServiceCreateForm.container.jsx [hunk 6]
```diff
@@ -0,0 +1,30 @@
+import React from "react";
```
Comment (ZERO-94) [inline src/containers/CustomerServiceCreateForm/CustomerServiceCreateForm.container.jsx:1]: is this container worth ? actually you only need to put all in the  CreateStaffModal Container and then check which tab is using to call the right API tbh

File: src/containers/DentistCreateForm/DentistCreateForm.container.jsx [hunk 7]
```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { dentistCreateAccount } from "../../services/teeth-apis/RegisterController";
+import { notification } from "antd";
+
+import DentistCreateFormComponent from "../../components/forms/DentistCreateFormComponent/DentistCreateForm.component";
+import DentistFormValueToDentistData from "../../mapper/DentistFormValueToDentistData";
+
+const DentistCreateFormContainer = () => {
```
Comment (ZERO-94) [inline src/containers/DentistCreateForm/DentistCreateForm.container.jsx:8]: is this container worth ? actually you only need to put all in the CreateStaffModal Container and then check which tab is using to call the right API tbh

**Review by ZERO-94 — CHANGES_REQUESTED** [review]

File: src/components/customized-components/SearchAccountFormComponent/SearchAccountForm.component.jsx (hunk 1 above)
Comment (kuroneko1011) [reply src/components/customized-components/SearchAccountFormComponent/SearchAccountForm.component.jsx:53]: have not implemented reset function

File: src/components/customized-components/SearchAccountFormComponent/SearchAccountForm.component.jsx (hunk 2 above)
Comment (kuroneko1011) [reply src/components/customized-components/SearchAccountFormComponent/SearchAccountForm.component.jsx:6]: because the table container file is too long

**Pull request merged**